gtsfm/two_view_estimator.py

```diff
@@ -106,7 +112,9 @@ def create_computation_graph(
         camera_intrinsics_i2_graph: Delayed,
         im_shape_i1_graph: Delayed,
         im_shape_i2_graph: Delayed,
-        i2Ti1_expected_graph: Optional[Delayed] = None,
+        wTi1_expected_graph: Optional[Delayed] = None,
```

Comment:
For consistency purposes, can we call these gt_wTi1_graph instead of wTi1_expected_graph?

Reply:
same with mesh